### PR TITLE
#3188: added pop-up to choose the relevant item type

### PR DESCRIPTION
**Pull request by ssreeram-DD**
if matched_work_path_template is none then UI pop-up with item_types.
User can choose required item type to be ingested.

### CODE REVIEW

File: hooks/basic/collector.py [hunk 1]
```diff
@@ -90,6 +91,51 @@
 }
 
 
+class PopItemTypesListUI(object):
```
Comment (mbrainerd) [inline hooks/basic/collector.py]: Pop ==> Popup

Comment (ssreeram-DD) [reply hooks/basic/collector.py]: i will update

File: hooks/basic/collector.py [hunk 2]
```diff
@@ -201,6 +266,12 @@ def settings_schema(self):
                 }
             }
         )
+        schema["Item Types UI"] = {
+            "type": "bool",
+            "default_value": False,
+            "allows_empty": True,
+            "description": "popup ui if work_path_template is none and having multiple item_types."
```
Comment (mbrainerd) [inline hooks/basic/collector.py]: "Popup UI to allow the user to specify the Item Type if one cannot be determined procedurally."

Comment (ssreeram-DD) [reply hooks/basic/collector.py]: updated description and given pull request

**Pull request merged**